### ISSUE TITLE
Manage scope for pkg.name

### ISSUE THREAD

**Issue by cballesta** [neo9/n9-node-sonar-generate#1]
Hi,

Could you manage to automatically replace scope in the pkg.name by a valid string for sonar projectKey ?
Something like @neo9/n9-node-sonar-generate => neo9_n9-node-sonar-generate for example.
I don't want to fill the sonar.projectKey for each projects.
You would be very kind ;)

SonarQube Scanner error with "@neo9/n9-node-sonar-generate" value for projectKey :
```
ERROR: Error during SonarQube Scanner execution
ERROR: Validation of project reactor failed:
  o "@neo9/n9-node-sonar-generate" is not a valid project or module key. Allowed characters are alphanumeric, '-', '_', '.' and ':', with at least one non-digit.
ERROR: 
ERROR: Re-run SonarQube Scanner using the -X switch to enable full debug logging.
ERROR: SonarQube scanner exited with non-zero code: 1
```
Cheers mate !

**Comment by gaetansenn** [comment]
Hi @cballesta 

What about `@neo9/n9-node-sonar-generate` -> `neo9:n9-node-sonar-generate`?

**Comment by cballesta** [comment]
You read my mind, I hesitated with this format !
Go for it !

**Comment by Atinux** [comment]
Done!

Thanks @gaetansenn for this idea and @cballesta for reporting this issue.

**Comment by atinux** [comment]
Fixed in [v1.0.2](https://github.com/neo9/n9-node-sonar-generate/releases/tag/v1.0.2)

**Comment by cballesta** [comment]
Thanks !
